spec/jekyll-remote-theme/integration_spec.rb

```diff
@@ -1,33 +1,96 @@
 # frozen_string_literal: true
 
 RSpec.describe "Jekyll::RemoteTheme Integration" do
-  let(:theme) { "pages-themes/primer" }
-  let(:config_path) { File.join source_dir, "_config.yml" }
-  let(:index_path) { File.join dest_dir, "index.html" }
-  let(:index_contents) { File.read(index_path) }
-  let(:stylesheet_path) { File.join dest_dir, "assets", "css", "style.css" }
-  let(:args) do
+  attr_reader :output, :status
+
+  def config_path
+    File.join source_dir, "_config.yml"
+  end
+
+  def malicious_config_path
+    File.join source_dir, "_malicious_config.yml"
+  end
+
+  def args(config_path)
     [
       "bundle", "exec", "jekyll", "build", "--config", config_path,
-      "--source", source_dir, "--dest", dest_dir, "--verbose",
+      "--source", source_dir, "--dest", dest_dir, "--verbose", "--safe",
     ]
   end
-  before { reset_tmp_dir }
-  after { reset_tmp_dir }
 
-  it "builds with the remote theme" do
+  def build_site(config_path)
     Dir.chdir tmp_dir do
-      output, status = Open3.capture2e(*args)
-      output = output.encode("UTF-8",
+      @output, @status = Open3.capture2e(*args(config_path))
+      @output = @output.encode("UTF-8",
```

Comment:
Can we do `output` in a `let(:output)` block instead of this? `let` is generall preferred over instance variables in RSpec that I've seen.